.github/workflows/lint.yml

```diff
@@ -27,7 +28,21 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Super-linter
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run lint
+        run: npm run lint
+
+      - name: Run format
+        run: npm run format
```

Comment:
Those need to be run locally as well. I will push changes moving this into the build scripts.